data/processed/aps/aps_econ_active_stem_density_data.csv

```diff
@@ -0,0 +1,281 @@
+,year,nuts_id,nuts_year_spec,aps_econ_active_stem_density_data
```

Comment:
Please remove the index column here

Reply:
Fixed- removed index column from all datasets.